data/hai/unfettered 0 prologue.txt

```diff
@@ -339,12 +341,12 @@ mission "Unfettered Jump Drive Trading"
 		not "Wanderers: Jump Drive Source: done"
 		# There is no minimum amount of JD to sell (other than the first 3), except if you failed to prove yourself the first time.
 		# "Unfettered Jump Drive 4: failed" represents the amount of JD sold.
-		"unfettered: jd to sell" < "Unfettered Jump Drive 4: failed"
+		"unfettered: jd to sell" <= "Unfettered Jump Drive 4: failed"
 		random < 20 + 8 * "Unfettered Jump Drive 4: failed"
 	on offer
 		conversation
-			`The Unfettered are starting to know you by reputation for bringing jump drives to them. This time, one of them<hood> - an engineer, judging by the writing pad and equipment they have - is waiting for you at your hatch.`
-			`	Maybe they mean to trade something other than money with you. Do you want to bargain for another drive with them?`
+			`As you exit your ship to head to the spaceport you see that an Unfettered Hai<hood> is waiting for you at your hatch.`
```

Comment:
I think the second <hood> substitution needs to end with a comma, but I can't comment directly on that line here.

@tibetiroka

Reply:
Yeah, a comma would make it much better. Also, this seems to be the only place where the substitution is used, so it should be fine to just edit it.